main/models.py

```diff
@@ -263,28 +273,66 @@ def fees_to_cashflows(date, fees):
             for x in fees if x.date <= date]
 
 
-def revision_to_cashflows(rev, end_date):
+def revision_to_cashflows(rev, end_date, tenancy_begin_date = date(2015, 6, 1), tenancy_end_date = date(2015, 12, 13)):
```

Comment:
Hi, in my understanding, in order to compute partial rents, we do need those params. Reason: A rent revision can differ from a tenancy period..

test case: as soon as I find time. I know, I know, tests should come first ...